src/test/common/steps/pages.js

```diff
@@ -79,6 +79,10 @@ class Pages {
   async openPageDirect (pageName) {
     await this.pageMap[pageName](this.url)
   }
+
+  async startApplication () {
+    this.enterDOB.open(this.url)
```

Comment:
It seemed sensible to create a start application function to avoid multiple updates every time the first page is changed. This seemed like a sensible place as it already has access to the URL, but maybe it should be a function in common steps?